apps/communications/dialer/js/call_log.js

```diff
@@ -524,17 +524,15 @@ var CallLog = {
       }
     }
     primInfoMain.appendChild(bdi);
-
-    var retryCount = document.createElement('span');
-    retryCount.className = 'retry-count';
+    primInfo.appendChild(primInfoMain);
```

Comment:
Haven't noticed where we may need this empty element, am I missing something? I changed it because it occupies space even if not visible (because of ```-moz-margin-start: 0.5rem```). If it's needed I can use css-only solution like ```.retry-count:empty { display: none; }```